src/components/TablePeople/TablePeople.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[],
+  slug: string,
+};
+
+export const TablePeople: React.FC<Props> = ({ people, slug }) => {
+  const isSelected = (person: Person) => person.slug === slug;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = people
+            .find(parent => parent.name === person.motherName);
+          const father = people
+            .find(parent => parent.name === person.fatherName);
```

Comment:
this functionality will be executed every time when the state is changed, so it's bad for the performance
so, make sense to move it to the variables with useMemo